src/main/java/com/univocity/parsers/common/processor/RowListProcessor.java

```diff
@@ -90,4 +92,18 @@ public void processEnded(ParsingContext context) {
 	public String[] getHeaders() {
 		return headers;
 	}
+
+    /**
+     * @return list of {@link Row} objects, one for each row of the csv file.
+     */
+  	public List<Row> getRowObjects() {
```

Comment:
This should be implemented in its own class. It's not efficient to load a list of strings first, and then convert everything to the `Row` objects. Please move this to a class that implements the `RowProcessor` interface and builds the `Row` objects on the fly (i.e. when the `rowProcessed` method is called).